examples/serve/genai_perf_client.sh

```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/env bash
+
+genai-perf profile \
+    -m TinyLlama-1.1B-Chat-v1.0 \
+    --tokenizer TinyLlama/TinyLlama-1.1B-Chat-v1.0 \
+    --service-kind openai \
+    --endpoint-type chat \
+    --num-prompts 1 \
```

Comment:
Let's use a bit more, like 100.
```suggestion
    --num-prompts 100 \
```

Reply:
Would that be too heavy for CI workloads?

Reply:
Ah NVM

Reply:
We can keep it as an example usage, users will need to select one of the following anyway: offline/concurrency/req rate.

Reply:
But this may cause some confusion when using together with request-rate, user can see how many requests genai-perf sends in the report (see the following screenshot), and this number is not as same as num-prompts.
<img width="565" alt="image" src="https://github.com/user-attachments/assets/75d0f713-0e15-41e9-8107-b93eec522b6e" />

Reply:
I think we'd better remove it if we don't use any offline dataset in this example.

Reply:
@LinPoly Users are going to use it anyway, if you think this is going to confuse the users, we should report to genai-perf.

@ganeshku1 Can you help clarify?

Reply:
@LinPoly You’re right. `--num-prompts` doesn’t control the request count.
`--num-prompts` tells GAP how many unique prompts it should prepare.

For example, if `--request-count` is set to 70 and `--num-prompts` is 100, GAP will generate **100 unique prompts**, but only create **70 input payloads** — since request count limits how many are actually sent. It will just sample from those 100 prompts.

There’s also an alias for this flag: [--num-dataset-entries](https://github.com/triton-inference-server/perf_analyzer/tree/main/genai-perf#:~:text=%2D%2Dnum%2Ddataset%2Dentries%20%3Cint%3E%3A%20The%20number%20of%20unique%20payloads%20to%20sample%20from.%20These%20will%20be%20reused%20until%20benchmarking%20is%20complete.).
To reduce confusion for users, especially in examples or docs, you could consider using `--num-dataset-entries` instead of `--num-prompts`.

Reply:
Thanks, then we can use request-count to control it.